tests/sentry/api/endpoints/test_event_committers.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+
+from datetime import datetime
+from django.core.urlresolvers import reverse
+
+from sentry.testutils import APITestCase
+
+
+class EventCommittersTest(APITestCase):
+    def test_simple(self):
```

Comment:
We should definitely be testing the actual important path. This test isn't testing anything useful in your code. We need to test a response with some commit data.